### PR TITLE
fix(fix nextdeadline and ordering): fix nextdeadline and ordering

### PR DESCRIPTION
**Pull request by mauriciokin**
fix nextdeadline and ordering criteria

### CODE REVIEW

**Review by christopher-semmens — DISMISSED** [review]

File: packages/forms-web-app/src/views/projects/examination-timetable.njk [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
   {% if events | length %}
 		{% if confirmedStartOfExamination %}
-    	<p class="govuk-body-s">{{confirmedStartOfExamination}}</p>
+    	<p>{{confirmedStartOfExamination}}</p>
```
[the remark this answers is not in the document]
Comment (mauriciokin) [reply packages/forms-web-app/src/views/projects/examination-timetable.njk:43]: we talked about it during our pairing today the sizes are still the same

[the remark this answers is not in the document]
Comment (christopher-semmens) [reply packages/forms-web-app/src/views/projects/examination-timetable.njk:43]: @mauriciokin ah yes, I remember, I hadn't realised it was `govuk-body-s`, by default all p tags are set to `govuk-body`.

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**